# Geometry Information of Poles, Radars and Cameras

poles:
  pole_1:
    xyzrpy: [-0.415985705489921, 1.11290021179622, 0.481, 1.5707963267949, 0.0, 0.785398163397456]
  pole_2:
    xyzrpy: [-1.59824637808195, -0.790114867663065, 0.461, 1.5707963267949, 0.0, 0.0]
  pole_3:
    xyzrpy: [0.964315628564147, 0.200335452740542, 0.461, 1.5707963267949, 0.0, 0.0]
  pole_4:
    xyzrpy: [0.000974248361056473, 0.150707650139425, 0.480970397293368, 1.5707963267949, 0.0, 0.0]

cameras:
  camera_11:
    width:  640
    height: 480
    xyzrpy: [-0.33159636363535226, -0.0883799948475499, 0.7180063415966533, -1.5707963267949, 0.785398163397456, 0]
    w2c:    [1.0, 0.0, 0.0, 0.14275258975532895, 0.0, 0.0, -1.0, 0.39262000515244677, 0.0, 1.0, 0.0, -0.3707190213749293, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_12:
    width:  640
    height: 480
    xyzrpy: [0.035417733646562054, -0.0800003106380456, 0.8332431804789487, -1.5707963267949, 2.35619449019234, 0]
    w2c:    [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.4009996893619487, 0.0, 0.0, 0.0, 0.19825031743993776, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_13:
    width:  640
    height: 480
    xyzrpy: [0.261482246581381, -0.040821726737309, 0.516250477506555, -1.5707963267949, -2.356194490192345, 0]
    w2c:    [0.0, 1.0, 0.0, -0.5487524281378586, 0.0, 0.0, -1.0, 0.4009996893619487, -1.0, 0.0, 0.0, 0.19825031743993776, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_14:
    width:  640
    height: 480
    xyzrpy: [-0.09284049268553585, -0.0420123457579493, 0.2698285834803398,-1.570796326794895, -0.785398163397456, 0]
    w2c:    [0.0, -1.0, 0.0, 0.8564544487126717, 0.0, 0.0, -1.0, 0.43898765424204556, 1.0, 0.0, 0.0, 0.2908362262996539, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_2:
    width:  640
    height: 480
    xyzrpy: [0.298406060019312, -0.0608075188884471, 0.460567976723035, 3.14159265358979, 1.5707963267949, -1.5707963267949]
    w2c:    [1.0, 0.0, 0.0, 1.2998403180626381, 0.0, 0.0, -1.0, 0.40019248111155975, 0.0, 1.0, 0.0, 1.2506828443860971, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_3:
    width:  640
    height: 480
    xyzrpy: [-0.00819043746300863, -0.0558609225821797, 0.201782124653999, -1.570796326794895, 1.570796326794895, 0]
    w2c:    [1.0, 0.0, 0.0, -0.9561251911011377, 0.0, 0.0, -1.0, 0.4051390774178211, 0.0, 1.0, 0.0, 0.001446671913453823, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_41:
    width:  640
    height: 480
    xyzrpy: [-0.00110365078598126, -0.145715673177097, 1.25963835404248, 0, -1.570796326794895, -1.570796326794895]
    w2c:    [-1.0, 0.0, 0.0, -0.00012940242492424545, 0.0, 0.0, -1.0, 0.33525472411627233, 0.0, -1.0, 0.0, -1.1089307039030527, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_42:
    width:  640
    height: 480
    xyzrpy: [0.0929564555419943, -0.145715673177097, 1.15083705256435, -1.5707963267949, 0, 0]
    w2c:    [0.0, -1.0, 0.0, -1.0001294024249248, 0.0, 0.0, -1.0, 0.335254724116267, 1.0, 0.0, 0.0, -0.09393070390305078, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_43:
    width:  640
    height: 480
    xyzrpy: [-0.000844845936131017, -0.145715673177097, 1.05677694623637, -1.570796326794895, 1.570796326794895, 0]
    w2c:    [1.0, 0.0, 0.0, -0.00012940242492637808, 0.0, 0.0, -1.0, 0.3352547241162705, 0.0, 1.0, 0.0, 0.9060692960969434, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_44:
    width:  640
    height: 480
    xyzrpy: [-0.0949049522641072, -0.145715673177097, 1.1505782477145, -1.570796326794895, 3.14159265358979, 0]
    w2c:    [0.0, 1.0, 0.0, 0.999870597575074, 0.0, 0.0, -1.0, 0.3352547241162689, -1.0, 0.0, 0.0, -0.09393070390304749, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_5:
    width:  640
    height: 480
    xyzrpy: [-1.4, 2.0, 0.7, 0.0, 0.0, 0.792156862745098]
    w2c:    [0.7118697166789228, -0.7023115451674349, 0.0, 2.4012406936853616, 0.0, 0.0, -1.0, 0.7, 0.7023115451674348, 0.7118697166789228, 0.0, -0.44050327012343715, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_6:
    width:  640
    height: 480
    xyzrpy: [1.1, 1.9, 0.7, 0.0, 0.0, 2.356194490192345]
    w2c:    [0.7071067811865476, 0.7071067811865475, 0.0, -2.1213203435596424, 0.0, 0.0, -1.0, 0.7, -0.7071067811865475, 0.7071067811865476, 0.0, -0.5656854249492381, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]
  camera_7:
    width:  640
    height: 480
    xyzrpy: [-0.75, 2.7, 0.7, 0.0, 0.0, 0.3490658503988659]
    w2c:    [0.34202014332566866, -0.9396926207859084, 0.0, 2.7936851836162044, 0.0, 0.0, -1.0, 0.7, 0.9396926207859083, 0.3420201433256687, 0.0, -0.21868492138987433, 0.0, 0.0, 0.0, 1.0]
    c2p:    [504.78216005824515, 0.0, 320.5, -0.0, 0.0, 504.78216005824515, 240.5, 0.0, 0.0, 0.0, 1.0, 0.0]
    R:      [36, 0, 0, 36]

radars:
  radar_2:
    offset: [-1.374475982753464, -1.1757576992522991, 0.461]
    angle:  90
    R:      [10, 0, 0, 10]  # [10.0, 0.0, 0.0, 0.0, 10.0, 0.0, 0.0, 0.0, 100.0]
  radar_3:
    offset: [0.9728163379849798, -0.0074961034847569885, 0.461]
    angle:  90
    R:      [10, 0, 0, 10]  # [10.0, 0.0, 0.0, 0.0, 10.0, 0.0, 0.0, 0.0, 100.0]

scene:
  kalman:
    model_size:   2
    A:            [1, 0, 0, 1]  # [1, 0, 1, 0, 0, 1, 0, 1, 0, 0, 1, 0, 0, 0, 0, 1]
    Q:            [1, 0, 0, 1]  # [10, 0, 0, 0, 0, 10, 0, 0, 0, 0, 100, 0, 0, 0, 0 ,100]
    max_age:      3
    iou_thres:    0.6
    scene_thres:  0.5
